### PR TITLE
Fix Bot Users Page

### PR DESCRIPTION
**Pull request by samcrichard**
Found a bunch of typos and grammar issues. Added information about when to use them and create them and added to PAYGO plans.

### CODE REVIEW

File: docs/user-management/index.md [hunk 1]
```diff
@@ -35,42 +39,33 @@ Bot Users differ from Users in a few important ways:
...
-  - Can be deleted to immediately revoke and delete all credentials associated with it
+Bot Users facts and limitations:
 
-### What is a Bot User?
-
-A Bot User is a service account that owns a set of credentials - authtokens, API keys, and SSH keys - independently of a person. You can see and manage your bot users in the [ngrok Dashboard](https://dashboard.ngrok.com/users/bots).
+- Bot users may be attached to services that consume ngrok platform resources and can trigger overages. These areas include metrics like bandwidth, total sessions and endpoints, auth tokens, API tokens, and more.
+- Bot users have limited functionality:
+  - Bot users cannot log into the ngrok Dashboard
+  - Bot users bannot be assigned or shared among multiple accounts
```
Comment (aaronshekey) [inline docs/user-management/index.md:50]:
```suggestion
  - Bot users cannot be assigned or shared among multiple accounts
```

File: docs/user-management/index.md [hunk 2]
```diff
@@ -35,42 +39,33 @@ Bot Users differ from Users in a few important ways:
...
+- Benefits of bot users:
+  - Credentials that are unique to a specific service or function can be connected to a bot user instead of being connected to a person. A standard user may leave the account or want to rotate their credentials, and these actions should not impact production services running in ngrok.
+  - Events are attributed to a Bot User and can help you better understand what a specific production service in ngrok is doing, even when there are multiple production services in the same account.
+  - Bot users can be deactivated to temporarily suspend all credentials associated with it--making ngrok more secure.
```
Comment (aaronshekey) [inline docs/user-management/index.md:54]:
```suggestion
  - Bot users can be deactivated to temporarily suspend all credentials associated with it—making ngrok more secure. 
```

File: docs/user-management/index.md [hunk 3]
```diff
@@ -35,42 +39,33 @@ Bot Users differ from Users in a few important ways:
...
+- Bot users may be attached to services that consume ngrok platform resources and can trigger overages. These areas include metrics like bandwidth, total sessions and endpoints, auth tokens, API tokens, and more.
+- Bot users have limited functionality:
+  - Bot users cannot log into the ngrok Dashboard
+  - Bot users bannot be assigned or shared among multiple accounts
+- Benefits of bot users:
+  - Credentials that are unique to a specific service or function can be connected to a bot user instead of being connected to a person. A standard user may leave the account or want to rotate their credentials, and these actions should not impact production services running in ngrok.
+  - Events are attributed to a Bot User and can help you better understand what a specific production service in ngrok is doing, even when there are multiple production services in the same account.
+  - Bot users can be deactivated to temporarily suspend all credentials associated with it--making ngrok more secure. 
+  - Bot users can be deleted to immediately revoke and delete all credentials associated with it--making ngrok more secure.
```
Comment (aaronshekey) [inline docs/user-management/index.md:55]:
```suggestion
  - Bot users can be deleted to immediately revoke and delete all credentials associated with it—making ngrok more secure.
```

File: docs/user-management/index.md [hunk 4]
```diff
@@ -35,42 +39,33 @@ Bot Users differ from Users in a few important ways:
 
 - Bot Users may not log into the Dashboard
 - Bot Users may not be a member of more than one Account
-- If your Bot Users do not counta
-
-Bot Users may be
+- Bot users do not count against any seat counts or limits.
+- Bot users can only be leveraged in ngrok Pay-as-you-go plans.
```
Comment (aaronshekey) [inline docs/user-management/index.md:43]:
```suggestion
- Bot users can only be leveraged in ngrok pay-as-you-go plans. 
```

Further down you don't capitalize `pay-as-you-go`. We should be consistent.

Comment (aaronshekey) [inline docs/user-management/index.md:43]:
```suggestion
- Bot users can only be used in ngrok Pay-as-you-go plans. 
```
Just a small peeve.

**Review by aaronshekey — CHANGES_REQUESTED** [review]

**Review by aaronshekey — APPROVED** [review]
🙌

**Review by nijikokun — APPROVED** [review]

**Pull request merged**